scylla/templates/scylla-statefulset.yaml

```diff
@@ -58,9 +62,15 @@ spec:
             mountPath: /opt/ready-probe.sh
             subPath: ready-probe.sh
       volumes: # TODO: data is discarded if pod restarts, change to something more persistant 
-        - name: scylla-data 
+        - name: scylla-data
+          {{- if .Values.statefulset.volume }}
```

Comment:
I should have said this before, but this is not a volume, but a host path. I would use `hostPath` as the name instead of volume

Reply:
done